portfolio/src/main/webapp/dairyChartScript.js

```diff
@@ -0,0 +1,122 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/** Fetches yearly dairy consumption data and uses it to create a chart. */
+async function drawDairyChart() {
+  const data = createChartDataFromDataset('consumption');
+  let options = createChartsOptions();
+
+  let properties = {
+    title: 'Dairy Consumption',
+    vAxis: { 
+      format: 'decimal',
+      title: 'Consumption (millions of pounds)'
+    }
+  };
+  // Add all custom properties to the options object
+  for(let p in properties) options[p] = properties[p];
```

Comment:
Look into using [`Object.assign`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign) or object [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#Spread_in_object_literals) (only available in modern browsers) to copy properties from one object to another. With modern JS there is generally a better alternative to `for...in` loops.

Reply:
Didn't know this syntax existed!